net/wallet/Portfile

```diff
@@ -0,0 +1,250 @@
+                    PERL=${prefix}/bin/perl${perl5.major} 
+
+# The Wallet client is just a couple of binaries, plus man pages and docs.
+subport wallet-client {
+    long_description-append The wallet client uses the remctl protocol to \
+                            communicate with a wallet server.
+
+    # We have our own destroot process, to install just client bits.
+    destroot {
+        # First, install common files from the distribution
+        xinstall -d ${destroot}${prefix}/share/doc/wallet-client
+        xinstall -m 644 ${worksrcpath}/LICENSE \
+            ${worksrcpath}/NEWS \
+            ${worksrcpath}/README \
+            ${worksrcpath}/TODO \
+            ${destroot}${prefix}/share/doc/wallet-client
+
+        # Install the wallet client executables
+        # ${destroot}${prefix}/bin already exists
+        xinstall -m 755 ${worksrcpath}/client/wallet \
+            ${worksrcpath}/client/wallet-rekey \
+            ${destroot}${prefix}/bin
+
+        # Install the wallet client man pages
+        xinstall -d ${destroot}${prefix}/share/man/man1
+        xinstall -m 644 ${worksrcpath}/client/wallet.1 \
+            ${worksrcpath}/client/wallet-rekey.1 \
+            ${destroot}${prefix}/share/man/man1
+    }
+}
+
+# The Wallet server is entirely Perl, spawned by the remctl daemon.
+# TODO: Patch wallet remctl config files to use ${destroot}${prefix}
+subport wallet-server {
+    # We can't use perl5.setup because it overrides alot of settings that
+    # we need.
+    #perl5.setup		Wallet 1.003
+
+    long_description-append The wallet server, run by remctld, maintains \
+                            the database of object metadata and secure \
+                            objects, and responds to requests from the \
+                            wallet client.
+
+    if {${perl5.major} != ""} {
+        depends_lib-append      port:p${perl5.major}-datetime \
+                                port:p${perl5.major}-dbi \
+                                port:p${perl5.major}-dbix-class \
+                                port:p${perl5.major}-digest-md5 \
+                                port:p${perl5.major}-sql-translator \
+                                port:p${perl5.major}-timedate
+        depends_build-append    port:p${perl5.major}-crypt-generatepassword \
+                                port:p${perl5.major}-datetime-format-sqlite
+    }
+
+    # TODO: Tests have a Stanford-specific part, and a NetDB-verifier part,
+    # which need to be disabled.
+    # test.run      yes
+    # test.target   check
+
+    # We have our own destroot process, to install just server bits.
+    destroot {
+        # First, install common files from the distribution
+        xinstall -d ${destroot}${prefix}/share/doc/wallet-server
+        xinstall -m 644 ${worksrcpath}/LICENSE \
+            ${worksrcpath}/NEWS \
+            ${worksrcpath}/README \
+            ${worksrcpath}/TODO \
+            ${destroot}${prefix}/share/doc/wallet-server
```

Comment:
If you don't want to repeat the name of the subport in the documentation path, you can use the `${subport}` variable.